src/Table.spec.tsx

```diff
@@ -0,0 +1,232 @@
+/**
+ * @jest-environment jsdom
+ */
+import React from "react";
+import { render, cleanup } from "@testing-library/react";
+import "@testing-library/jest-dom/extend-expect";
+import { Table } from "./Table";
+
+interface User {
+  name: string;
+  email: string;
+  image: string;
+  dateAdded: string;
+}
+
+const users: ReadonlyArray<User> = [
+  {
+    name: "Alice Howell",
+    email: "alice@starkindustries.net",
+    image: require("./table.story/alice-howell.png"),
+    dateAdded: "05/25/2019",
+  },
+  {
+    name: "Benjamin Lawrence",
+    email: "Ben@starkindustries.net",
+    image: require("./table.story/benjamin-lawrence.png"),
+    dateAdded: "05/26/2019",
+  },
+  {
+    name: "Cynthia Bowman",
+    email: "cbowman@starkindustries.net",
+    image: require("./table.story/cynthia-bowman.png"),
+    dateAdded: "05/27/2019",
+  },
+  {
+    name: "Jeremy Jacobs",
+    email: "jj@starkindustries.net",
+    image: require("./table.story/jeremy-jacobs.png"),
+    dateAdded: "05/28/2019",
+  },
+  {
+    name: "Jeremy Griffin",
+    email: "jgriffin@starkindustries.net",
+    image: require("./table.story/jeremy-griffin.png"),
+    dateAdded: "05/29/2019",
+  },
+];
+
```

Comment:
once this is merged, perhaps @fmvfelipemonsalve should consider adding tests for the width passing?

Reply:
Yes :) When new functionality is added, tests should be added to prove that it works.